etc/kurtosis.yaml

```
@@ -3,10 +3,25 @@ ethereum_package:
     - el_type: reth
       cl_type: lighthouse
 optimism_package:
-  participants:
-    - el_type: op-reth
-      el_image: ghcr.io/ithacaxyz/odyssey:latest
-      count: 2
-  network_params:
-    network_id: "41144114"
-    seconds_per_slot: 1
+  chains:
+    - participants:
+      - el_type: op-reth
+        el_image: ghcr.io/ithacaxyz/odyssey:latest
+        cl_image: ghcr.io/paradigmxyz/op-node:latest
+        el_extra_env_vars:
+          "EXP1_SK": "0x8b3a350cf5c34c9194ca85829a2df0ec3153be0318b5e2d3348e872092edffba"
+          "EXP1_WHITELIST": "0x0116686E2291dbd5e317F47faDBFb43B599786Ef"
```

nit, think the current format is not normal in yaml

```suggestion
          EXP1_SK: "0x8b3a350cf5c34c9194ca85829a2df0ec3153be0318b5e2d3348e872092edffba"
          EXP1_WHITELIST: "0x0116686E2291dbd5e317F47faDBFb43B599786Ef"
```